quill-finagle-postgres/src/main/scala/io/getquill/context/finagle/postgres/FinaglePostgresEncoders.scala

```diff
@@ -11,14 +11,14 @@ import org.jboss.netty.buffer.ChannelBuffers
 trait FinaglePostgresEncoders {
   this: FinaglePostgresContext[_] =>
 
-  protected case class ValueEncoderEncoder[T](encoder: ValueEncoder[T]) extends Encoder[T] {
+  case class ValueEncoderEncoder[T](encoder: ValueEncoder[T]) extends Encoder[T] {
```

Comment:
As one motivating use case I plan, in my code, to provide a macro implicit materializer which provides a ValueEncoderEncoder for any type with an implicit ValueEncoder. Doing so requires me to be able to name the materializer macro's return type.